ehrapy/preprocessing/_bias.py

```diff
@@ -85,12 +93,21 @@ def detect_bias(
         sens_features_list = adata.var_names.values.tolist()
         cat_sens_features = adata.var_names.values[adata.var[FEATURE_TYPE_KEY] == CATEGORICAL_TAG]
     else:
+        sens_features_list = []
         for feat in sensitive_features:
             if feat not in adata.var_names:
-                raise ValueError(f"Feature {feat} not found in adata.var.")
-        sens_features_list = sensitive_features
+                # check if feature has been encodeds
+                encoded = [f for f in adata.var_names if f.startswith(f"ehrapycat_{feat}")]
+
+                if len(encoded) == 0:
+                    raise ValueError(f"Feature {feat} not found in adata.var.")
```

Comment:
I changed it to `variable` now, because, otherwise, I'd have a naming conflict with the `feature` variable from your previous comment. But yes, we'll discuss again tomorrow ^^